src/Microsoft.Sbom.Parsers.Spdx30SbomParser/Entities/Element.cs

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections;
+using System.Collections.Generic;
+using System.Text.Json.Serialization;
+
+namespace Microsoft.Sbom.Parsers.Spdx30SbomParser.Entities;
+
+/// <summary>
+/// Base domain class from which all other SPDX-3.0 domain classes derive.
+/// https://spdx.github.io/spdx-spec/v3.0.1/model/Core/Classes/Element/
+/// </summary>
+public class Element
```

Comment:
Should this be abstract? I'm not sure if we ever have just an `Element` that isn't a derived object.

Reply:
Yes it can be made abstract, there will be no objects other than derived ones.